src/components/PluginVersions/PluginVersionsTable.tsx

```diff
@@ -0,0 +1,81 @@
+                                        <th>{parsedData.name}</th>
+                                        {parsedData.pluginVersions.map((version: string) => (
+                                            <th key={version}>{version}</th>
+                                        ))}
+                                        <th>Sum</th>
+                                    </tr>
+                                </thead>
+                                <tbody>
+                                    {parsedData.rows.map((row) => (
+                                        <tr
+                                            key={row.coreVersion}
+                                            className={row.coreVersion.includes('lts') ? 'lts' : ''}
+                                        >
+                                            <th>{row.coreVersion}</th>
+                                            {row.cells.map((cell) => (
+                                                <td
+                                                    key={cell.version}
+                                                    title={cell.title}
+                                                    style={{ opacity: cell.opacity }}
+                                                >
+                                                    {cell.count > 0 ? cell.count : ''}
+                                                </td>
+                                            ))}
+                                            <td className="subtotal" title={row.totalTitle}>
+                                                {row.total}
+                                            </td>
+                                        </tr>
+                                    ))}
+                                </tbody>
+                                <tfoot>
+                                    <tr>
+                                        <th>Total</th>
+                                        {parsedData.pluginVersions.map((version: string) => (
+                                            <td key={version}>
+                                                {parsedData.rows.reduce(
+                                                    (acc: number, row) =>
+                                                        acc + row.cells.find((cell) => cell.version === version)!.count,
+                                                    0
+                                                )}
+                                            </td>
+                                        ))}
+                                        <td>{parsedData.totalInstalls}</td>
+                                    </tr>
+                                </tfoot>
+                            </table>
+                        </div>
+                    )
+                )
+            ) : (
+                <>
+                    <h1>Plugin Versions</h1>
+                    <p>Click on a plugin to view its versions</p>
```

Comment:
```suggestion
                    <p>Please click on a plugin in the list available from the side navbar to view its versions</p>
```

Reply:
Maybe we can use MUI Typography for handling text like in L73-74?: https://mui.com/material-ui/api/typography/

Reply:
Hi @krisstern 

I say we keep it as is for now (I changed the text) and we make the changes on the future PR when we remove sidebar and instead use a dropdown selector as discussed.